platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/network/NetworkUtils.java

```diff
@@ -129,38 +128,31 @@ public static String formatException(final Throwable e) {
     }
 
     /**
-     * Create a {@link SocketFactory} based on the configuration and the provided keys and certificates. NOTE: This
-     * method is a stepping stone to decoupling the networking from the platform.
+     * Create a TLS-based {@link SocketFactory} using the provided keys and certificates.
+     * NOTE: This method is a stepping stone to decoupling the networking from the platform.
      *
      * @param selfId        the ID of the node
-     * @param addressBook   the address book of the network
+     * @param peers         the list of peers
      * @param keysAndCerts  the keys and certificates to use for the TLS connections
      * @param configuration the configuration of the network
      * @return the created {@link SocketFactory}
      */
     public static @NonNull SocketFactory createSocketFactory(
             @NonNull final NodeId selfId,
```

Comment:
This method no longer does much but the error handling is still very tidy to have here, so I've made the decision to keep it it.